### DISCUSSION TITLE
Start Finishing (Sept/Oct 2022)

### DISCUSSION THREAD

**Discussion by tomcudd** [Virtual-Coffee/virtualcoffee.io#626] [VC Book Club]
### Start Finishing
Each week near the end of our reading session, I'll post our discussion questions and points in this thread and also provide a link in Slack. Please feel free to continue the conversation in the forum of your choice.

### Part 1 (Clearing the Decks for Your Best Work)
- Aug 29-Sept 8: Chapter 1 ("Someday" Can Be Today) + 2 (Getting to Your Best Work)
- Sept 9-Sept 15: Chapter 3 (Pick an Idea That Matters to You)

### Part 2 (Planning Your Project)
- Sept 16-Sept 22: Chapter 4 (Convert Your Idea into a Project)
- Sept 23-Sept 29: Chapter 5 (Make Space for Your Project)
- Sept 30-Oct 6: Chapter 6 (Build Your Project Road Map)
- Oct 7-Oct 13: Chapter 7 (Keep Flying by Accounting for Drag Points)

### Part 3 (Working the Plan)
- Oct 14-Oct 20: Chapter 8 (Weave Your Project into Your Schedule)
- Oct 21-Oct 27: Chapter 9 (Build Daily Momentum) + 10 (Finish Strong)

**Comment by tomcudd** [comment]
### Aug 29-Sept 8: Chapter 1 ("Someday" Can Be Today) + 2 (Getting to Your Best Work)
**Chapter 1**
- Over the last 2 weeks how much time and attention have you focused on what matters to you?
- How many projects have you start and left unfinished in the last month or year?
- Do you know the think you should be working on and just aren't doing it?
- Do you consider work, personal, and home projects based on the definition in the book?
- What parts of the "best work" definition resonate with you?
- Have you seen a similar pattern of 3-5 year periods of propelling change in your life?
- Where are you struggling in your projects (comparing day to day beginnings, muddle middle, and ideal ends)?

**Chapter 2**
- Do you find that there's a gap in your current work and expectations on yourself for what you should be getting done?
- Which of the five challenges do you most find get in the way of your best work?
- Of the ways to overcome those challenges, which of the ideas seem possible for you to take on today?
- Do you find that the next step you know you should be doing stretches your comfort level and ability?
- How do you reinforce your positive habits that help you complete your projects?
- Do you have a clear next step in a project you're struggling to take?
- Can you say no to more things that are coming up?
- Which of the keys described can you incorporate into your working practices today?

**Comment by tomcudd** [comment]
### Sept 9-Sept 15: Chapter 3 (Pick an Idea That Matters to You)
- Do you find yourself thrashing/flailing/floundering on important work you know needs to get done?
- Are these things you find yourself thrashing on important projects to you?
- During what part of a project do you find yourself flailing?
- When do you find yourself falling back to simpler, easier tasks to work on?
- Do you feel emotional, physical pain when you are not doing your best work?
- Have you felt resentment towards people who are completing or seem capable of their best work?
- How does displacing the time you spend on one task over another affect you?
- How will you feel about letting projects and ideas go that you will eliminate from your lists?
- Have you completed the list creation methods described in the book to get rid of projects you will not work on?

**Comment by tomcudd** [comment]
### Part 2 (Planning Your Project)
### Sept 16-Sept 22: Chapter 4 (Convert Your Idea into a Project)
- If you haven't done the work to pick a project idea, what's impeding you from completing that?
- Does your project have enough distinct details to get started?
- Are you able to use the creative SMART goal method to form your road map and is it stated clearly so you know what to do next?
- Is it immediately clear why this project is important for you to accomplish?
- Do you have resources available to start the journey towards your goal?
- Can you change your expectations about your goal to make it more realistic to accomplish?
- Have you considered evaluating your levels of success to create different goal standards?
- Is your commitment and effort in line with your expectations for success?
- Who will be a part of your group that helps you succeed?

**Comment by tomcudd** [comment]
### Sept 23-Sept 29: Chapter 5 (Make Space for Your Project)
- Do you see patterns of 15 minute tasks and 2 hour blocks of work in your existing projects?
- Have you felt your attention/energy/focus wane at the end of those time periods?
- Does the project pyramid make a clearer visualization of how many large or ambitious projects you are doing at once? Do you have too many to be able to complete?
- Does the use of specific verbs in listing your work items help you define your time/effort associated with it?
- How can shifting your perspective within a project either make clearer the purpose or inform your next steps?
- How does limiting your active projects affect your "to do" list when deciding what to work on?
- Are you looking for how to fit focus blocks into your schedule?
- Do you batch administrative tasks in your day or work through them as interruptions come in?

**Comment by tomcudd** [comment]
### Sept 30-Oct 6: Chapter 6 (Build Your Project Road Map)
- What projects can you re-frame to take advantage of your strengths?
- Come up with a list of 12-15 GATES as described. Can you leverage any for your project?
- What methods can you build from your unique strengths and abilities?
- Have you created a project budget?
- Are you building your plan based on capacity or deadline?
- Are you driving your project on results or the process?
- Have you accounted for the relay time or external dependencies needed to accomplish your work?
- Have you started your project road map with the sequenced, sorted, linked chinks of work as described?
- Are the chunks in a timeline or scheduled?

**Reply by CuriousCurmudgeon** [reply]
I have very mixed feelings about the type of project road maps he talks about. It starts to feel a lot like Gantt charts, which I've rolled my eyes at throughout my career. At the same time, I think there is value in putting the thought into that type of planning as long as you're revisiting and adjusting on a regular basis. The author advocates for that as _momentum planning_ a bit later in the book.

Overall, my planning philosophy follows the Eisenhower quote of "plans are useless, but planning is everything" and I think the author follows the spirit of that.

**Comment by tomcudd** [comment]
### Oct 7-Oct 13: Chapter 7 (Keep Flying by Accounting for Drag Points)
- Do you tell yourself no-win scenarios that limit your success?
  - Do you see success negatively affecting relationships around you?
  - Do you believe your achievements will come at the expense of some virtue?
  - Do you think your previous success can't be replicated?
- Have you chosen a safe bet that is leading to mediocrity?
- How are other people's priorities slowing you down? What boundaries or accommodations can you enact to help prioritize your efforts?
- Can you use these outside priorities in a way that will help you? (Gain an ally, learn a skill, etc.)
- Is it possible to figure out how turn a derailer or naysayer into someone on your side?
- What would show up in your project "premortem" when you think about the ways it can go wrong?

**Comment by tomcudd** [comment]
### Part 3 (Working the Plan)
### Oct 14-Oct 20: Chapter 8 (Weave Your Project into Your Schedule)
- Are you working in an environment that maximizes your best efforts?
- What factors help you achieve the most when you consider noise, light, space, location in your work area?
- Are there times it is more effective to change locations rather than improve your current work space?
- What types of work do you find helpful to batch or stack?
- What are the dreadful work items that you may need to tackle in the first or early part of your days?
- Is there a type of day you're most effective and can you put your critical work then?
- Can you utilize different days of the week to take on tasks in different ways?
- Have you tried the 5/10/15 method described in the book to shift your planning to the end of your previous day?
- How can you use momentum planning to continually adjust your efforts as your project rolls along?

**Comment by tomcudd** [comment]
### Oct 21-Oct 27: Chapter 9 (Build Daily Momentum)
- Does working while in the flailing parts of a project benefit you?
- Are you goal or progress focused?
- How do you celebrate your wins and accomplishments?
- What routine or habit have you developed to build momentum?
- What serves as your "crumb trail" to make it simpler to get back to your work path?
- What do you do to quiet interruption and distraction?
- What causes you to get stuck in a project near the end? How do you overcome it?

**Comment by tomcudd** [comment]
### Oct 28-Nov 1: Chapter 10 (Finish Strong)
- Are there people you rely on who can remind you to take a victory lap when you've reached a goal?
- How do you plan to make space and time to recover in between your projects?
- What parts of cleanup do you often find yourself needing to work on when a project is completed?
- How do you decide what to archive or trash?
- Where can you record your after action review to help make improvements for your next project?
- What new possibilities are now unlocked with your finished project?
